### PR TITLE
default form-inline class applied to parent content div on date select helpers

### PR DESCRIPTION
**Pull request by lancecarlson**
can also override this behavior with skip_inline: true

### CODE REVIEW

**Comment by lancecarlson** [comment]
Fixes: https://github.com/bootstrap-ruby/bootstrap_form/issues/460

File: lib/bootstrap_form/form_builder.rb [hunk 1]
```diff
@@ -52,7 +52,9 @@ def initialize(object_name, object, template, options)
       define_method(with_method_name) do |name, options = {}, html_options = {}|
         prevent_prepend_and_append!(options)
         form_group_builder(name, options, html_options) do
-          content_tag(:div, send(without_method_name, name, options, html_options), class: control_specific_class(method_name))
+          html_class = control_specific_class(method_name)
+          html_class = "#{html_class} form-inline" unless options[:skip_inline]
```
Comment (lcreid) [inline lib/bootstrap_form/form_builder.rb]: I suspect that you will need to be more specific here about when to add `form-inline`. I believe you only want to add it when the `form_layout` is `:horizontal`.

Comment (lcreid) [inline lib/bootstrap_form/form_builder.rb]: Is the `:skip_inline` option a new option you're proposing? If so, could you please update the CHANGELOG to note the new functionality, and probably add a description of the option in the `README`.

**Comment by lancecarlson** [comment]
@lcreid I have no problem writing tests, but I'm having a hard time getting them to run. It keeps saying this:

ActiveRecord::StatementInvalid: Could not find table 'users'

**Comment by lcreid** [comment]
Yes. I just figured that out myself. I'll fix the documentation.

**Comment by lcreid** [comment]
Documentation fix in PR #462.

**Comment by lancecarlson** [comment]
@lcreid I updated the changelog in this latest commit, but I'm not entirely sure where the skip_inline on the field helper should be documented. I only put it there so people could override if that wasn't the behavior they wanted.

**Comment by lcreid** [comment]
Sorry for not following up on this sooner! This is looking good, but it occurs to me that we should do the same for `time_select` and `datetime_select`. Would you mind adding those to this PR similarly to the way you did `date_select`? Sorry I didn't mention this sooner.

**Comment by lcreid** [comment]
@lancecarlson Please let me know if you're going to add this functionality for `time_select` and `datetime_select`. I can do it if you don't have time. Just let me know, please. Thanks in advance!

**Comment by lcreid** [comment]
@lancecarlson thanks for the PR! My apologies for not noticing sooner that you had extended it to cover all the date and time selects. I resolved the conflicts.

@mattbrictson can you please take a quick look at this to make sure the PR doesn't add functionality that it shouldn't? The Bootstrap 4 version was stacking the fields for date and time, rather than putting them in the more intuitive side-by-side #460.

**Pull request merged**